_includes/solver-tables.html

```diff
@@ -305,10 +305,18 @@
       <td>Comm.</td> <!-- License -->
     </tr>
 
-
-
-
-
+    <tr>
+      <th>
+        <a href="http://scip.zib.de/">SCIP</a>
+        <a href="https://github.com/SCIP-Interfaces/SCIP.jl">(.jl)</a>
+      </th>
+      <td>&#10004;</td> <!-- Linear/Quadratic -->
+      <td></td> <!-- Convex conic -->
```

Comment:
Add a line like [this](https://github.com/JuliaOpt/CPLEX.jl/blob/a07d7099b92753a876a12ddede2265b679550cbe/src/CplexSolverInterface.jl#L33) and you're all set.